plugins/karma.py

```diff
@@ -118,14 +123,15 @@ def points(text, chan, db, conn):
                 pos += int(k[0])
             score += int(k[0])
         if thing:
-             return "{} has a total score of {} (+{}/{}) across all channels I know about.".format(thing, score, pos, neg)
+            return "{} has a total score of {} (+{}/{}) across all channels I know about.".format(thing, score, pos, neg)
         return "{} has a total score of {} (+{}/{}) in {}.".format(text, score, pos, neg, chan)
     else:
         return "I couldn't find {} in the database.".format(text)
 
+
 @hook.command("topten", "pointstop", "loved", autohelp=False)
 def pointstop(text, chan, db, message, conn, notice):
-    """.topten or .pointstop prints the top 10 things with the highest points in the channel. To see the top 10 items in all of the channels the bot sits in use .topten global."""
+    """.topten or .pointstop or .loved prints the top 10 things with the highest points in the channel. To see the top 10 items in all of the channels the bot sits in use .topten global."""
```

Comment:
If you replace this with `"""- prints the top 10 things with the highest points in the channel. To see the top 10 items in all of the channels the bot sits in use .topten global."""`, cloudbot will automatically add the command and command prefix before the documentation, instead of having to hardcode it